modules/ui/ModelTab.py

```diff
@@ -235,8 +235,8 @@ def __create_base_components(
 
         if has_vae:
             # base model
-            components.label(self.scroll_frame, row, 0, "VAE",
-                             tooltip="Directory or Hugging Face repository of a VAE model. Can be used to override the base model VAE.")
+            components.label(self.scroll_frame, row, 0, "VAE (Read Tooltip)",
```

Comment:
I'm not sure if "(Read Tooltip)" is a good addition to the name. Generally, all tooltips should be read. A better solution (in my opinion) would be to add a more descriptive error message when the VAE loading fails.

Reply:
It seems error reporting is done with this line at the end of a model loader? It seems difficult to tailor that to a specific error case.
raise Exception("could not load model: " + model_names.base_model)